pull/github_context.go

```diff
@@ -236,7 +241,7 @@ func (ghc *GithubContext) CurrentSuccessStatuses(ctx context.Context) ([]string,
 			}
 
 			for _, s := range checkRuns.CheckRuns {
-				if s.GetConclusion() == "success" {
+				if _, allowed := allowedCheckConclusions[s.GetConclusion()]; allowed {
```

Comment:
If `allowedCheckConclusions` was a `map[string]bool`, this could be simplified to:

```go
if allowedCheckConclusions[s.GetConclusion()] {
    // ...
}
```

Reply:
The conditional is definitely easier to read this way. Done!